installer/templates/phx_web/telemetry.ex

```diff
@@ -0,0 +1,52 @@
+defmodule <%= web_namespace %>.Telemetry do
+  use Supervisor
+  import Telemetry.Metrics
+
+  def start_link(arg) do
+    Supervisor.start_link(__MODULE__, arg, name: __MODULE__)
+  end
+
+  def init(_arg) do
+    children = [
+      {:telemetry_poller, measurements: periodic_measurements(), period: 10_000}
+      # Add reporters as children of your supervision tree.
+      # {Telemetry.Metrics.ConsoleReporter, metrics: metrics()}
+    ]
+
+    Supervisor.init(children, strategy: :one_for_one)
+  end
+
+  defp metrics do
+    [
+      # Phoenix Metrics
+      summary("phoenix.endpoint.stop.duration",
+        unit: {:native, :millisecond}
+      ),
+      summary("phoenix.router_dispatch.stop.duration",
+        tags: [:route],
+        unit: {:native, :millisecond}
+      ),<%= if ecto do %>
```

Comment:
@mcrumm I changed the metrics we emit here. Doing a tag per request path led to an explosition of data when trying it in prod, so I removed that metric. Doing per controller+action was not helpful for LiveView, so I am doing it per route instead.

Reply:
Good call!